src/consumer/batch.js

```diff
@@ -23,4 +23,13 @@ module.exports = class Batch {
           .toString()
       : this.messages[this.messages.length - 1].offset
   }
+
+  offsetLag() {
+    return this.isEmpty()
+      ? '0'
+      : Long.fromValue(this.highWatermark)
+          .add(-1)
```

Comment:
Why was this -1 necessary again? I think it makes sense to make this a named constant to avoid having to ask this question again.

Reply:
`highWatermark` has the next possible offset. The last offset of the topic is `highWatermark - 1`